frontend/index.html

```diff
@@ -725,16 +792,18 @@ <h2 data-i18n="contact">Contact</h2>
       $(document).ready(function() {
         STUDY_NAME = _getStudyName(window.location);
         const CONFIG_URL = `https://raw.githubusercontent.com/e-mission/nrel-openpath-deploy-configs/main/configs/${STUDY_NAME}.nrel-op.json`;
-
         // set the initial value to the current language
         // otherwise, if the dropdown is set to es and we reload the page, the
         // text changes to en, but the dropdown stays as es
         // we use the non-changing version of the selection because we don't
         // want to redisplay the text twice
         // https://stackoverflow.com/a/16979926/4040267
+        
         $('#langSel').val(curr_lang);
-        const SUPPORTED_LANGUAGES = $.map($('#langSel')[0].options, (e) => e.value);
-        console.log("After reading from options, languages are ", SUPPORTED_LANGUAGES);
+        //assuming all languages in i18n folder are possible
+        //used to be all the langs from the menu, but that's dynamic now, and not loaded yet
+        SUPPORTED_LANGUAGES = ["en", "es", "lo"];
+        console.log("Based on the list, languages are ", SUPPORTED_LANGUAGES);
```

Comment:
I would remove this. You are correct that at this point, we don't have the list of languages. You can just have the dropdown disabled by default until we have read in the languages.

Reply:
removed this in [78c9c78](https://github.com/e-mission/nrel-openpath-join-page/pull/18/commits/78c9c786025c4398e4f15174b8ca4ac914441c0e)